### ISSUE TITLE
When New Architecture is enabled Bugfender throws `RNRnBugfenderSpec.h' file not found error on iOS

### ISSUE THREAD

**Issue by PunainenAurinko** [bugfender/rn-bugfender#65]
When you enable fabric and New Architecture in your RN app, RN Bugfender tries to import a file that does not exist and the iOS install process throws an error:

`RNRnBugfenderSpec.h' file not found (RnBugfender.mm:6:9)`

The error is caused by the following import statement in RnBugfender.mm file:

```
#ifdef RCT_NEW_ARCH_ENABLED
#import "RNRnBugfenderSpec.h"
#endif
```
If you disable this import the build process fails on `facebook::react` in the next block that is supposed to run in New Architecture:

```
#ifdef RCT_NEW_ARCH_ENABLED
- (std::shared_ptr<facebook::react::TurboModule>)getTurboModule:
(const facebook::react::ObjCTurboModule::InitParams &)params
{
    return std::make_shared<facebook::react::NativeRnBugfenderSpecJSI>(params);
}
#endif
```

Please advise what can be done if we want to enable RN New Architecture in our project and continue to use Bugfender at the same time. 

We're using:

```
"@bugfender/rn-bugfender": "^2.1.2",
"react-native": "0.71.3",
"react-native-reanimated": "^3.0.0",
"react-native-screens": "^3.20.0",
"react-native-gesture-handler": "^2.9.0",
```

Thank you!

**Comment by franmontiel** [comment]
Hi @PunainenAurinko we still do not support the new architecture in Bugfender but we plan to provide support for it in the near future.

I'll update this issue once we launch a new version with support for it

**Comment by ajeetfancode** [comment]
Ran into the same issue, we are migration our RN app to New architecture and even after adding the `RnBugfender` to `react-native.config.js` file, its giving me same error: `RNRnBugfenderSpec.h' file not found`
In `react-native.config.js` file
`project: {
    ios: {
      unstable_reactLegacyComponentNames:["RnBugfender"]
 }
}`
Any help on this

**Comment by jgimenez** [comment]
Fixed in [v2.2.0](https://github.com/bugfender/rn-bugfender/releases/tag/v2.2.0), now published.
